getter.py

```diff
@@ -1,29 +1,88 @@
+from os.path import exists
 import requests as r
 import json
 
 class Getter:
     def __init__(self):
         self.key = 'qhqws47nyvgze2mq3qx4jadt'
         self.skuId = None
-        self.productData = None
-    
+
     def getSKUFromURL(self, best_buy_url):
         return best_buy_url.split('skuId=')[1]
```

Comment:
Could you error check here? If the user types in a bad url or part of a best buy url that case should be handled. Other than that, it looks great!

Reply:
It was easier to add error check in the main.py. Once this pull request is merged I can create a new pull request with the updated main.py.